hierarchicalforecast/utils.py

```diff
@@ -255,9 +266,12 @@ def aggregate(
 
     #------------------------------- Aggregation -------------------------------#
     n_agg = S_df.shape[0] - S_df.shape[1]
-    Agg = S_df.values[:n_agg, :]
-    y_bottom = balanced_df.y.values
+    if sparse_s:
+        Agg = S_df.sparse.to_coo().tocsr()[:n_agg, :]
```

Comment:
Can you add a test in the notebook to verify that this is working as expected? Could be just a small example with sparse_s set to both True and False and verify that the outputs are the same.

Reply:
Done